mk_verificator/tests/networking/mtu/test_mtu.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "groups",
+    utils.get_groups(utils.get_configuration(__file__))
+)
+def test_mtu(local_salt_client, groups):
+    skipped_ifaces = ["bonding_masters", "lo"]
```

Comment:
please move to the config and use global config like
*import group*
test_config = utils.get_configuration(__file__)

def test_1():
    skipped_ifaces = config['skipped_ifaces']